gemfiles/rails_5.gemfile

```diff
@@ -0,0 +1,13 @@
+# This file was generated by Appraisal
+
+source "https://rubygems.org"
+
+gem "appraisal"
+gem "pry"
+gem "rake"
+gem "sqlite3"
```

Comment:
interesting that rails5 and 6 don't have the `~> 1.4` here in the generated files, should they?

Reply:
Good catch, I've run `bundle exec appraisal update` to update the generated gemfiles.